lib/cfn-nag/custom_rules/EC2NetworkAclEntryReusedPortsRule.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require 'cfn-nag/util/truthy'
+require_relative 'base'
+
+class EC2NetworkAclEntryReusedPortsRule < BaseRule
+  def rule_text
+    'NetworkACL Entries are reusing or overlapping ports which may create ineffective rules.'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W72'
+  end
+
+  def audit_impl(cfn_model)
+    violating_nacl_entries = []
+    cfn_model.resources_by_type('AWS::EC2::NetworkAcl').each do |nacl|
+      violating_nacl_entries += violating_nacl_entries(nacl)
+    end
+
+    violating_nacl_entries.uniq.map(&:logical_resource_id)
```

Comment:
is the use of uniq necessary and tested?  i don't believe hash or eql? are defined for the nacl entry model element?